### PR TITLE
bug fix: delete vertices processor may core, if this vertex also have edges and index

### PR DESCRIPTION
**Pull request by liuyu85cn**
get one core screen from BBS. 

https://discuss.nebula-graph.com.cn/t/topic/1123/8

check all reference to getTagId, 
find deleteVerticesProcessor call this without check if it is a vertex first.

### CODE REVIEW

**Review by dangleptr — APPROVED** [review]
Well done. LGTM. Good catch

**Review by critical27 — APPROVED** [review]

File: src/storage/mutate/DeleteVerticesProcessor.cpp [hunk 1]
```diff
@@ -95,6 +95,9 @@ DeleteVerticesProcessor::deleteVertices(GraphSpaceID spaceId,
         TagID latestVVId = -1;
         while (iter->valid()) {
             auto key = iter->key();
+            if (!NebulaKeyUtils::isVertex(key)) {
+                break;  // there isn't any version of curr vertex, go to next
+            }
```
Comment (panda-sheep) [inline src/storage/mutate/DeleteVerticesProcessor.cpp:100]:
```
   if (!NebulaKeyUtils::isVertex(key)) {
          iter->next();
          continue;
   }
```

**Review by panda-sheep — APPROVED** [review]
Excellent!

**Comment by bright-starry-sky** [comment]
A question about header : "delete vertices processor may core, if this vertex also have edges and index", where is index ? I think there are only edges. right?

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Pull request merged**